@xen-orchestra/lite/src/App/Pools/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Map } from 'immutable'
+import { Route, Switch } from 'react-router'
+import { withState } from 'reaclette'
+
+import PoolNetwork from './PoolNetwork'
+
+import { ObjectsByType, Pool } from '../../libs/xapi'
+
+interface ParentState {}
+
+interface State {
+  objectsByType: ObjectsByType
+}
+
+interface Props {}
+
+interface ParentEffects {}
+
+interface Effects {}
+
+interface Computed {
+  objectsFetched: boolean
+  pools?: Map<string, Pool>
+}
+
+const Pools = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
+  {
+    computed: {
+      objectsFetched: state => state.objectsByType !== undefined,
+      pools: state => (state.objectsFetched ? state.objectsByType?.get('pool') : undefined),
+    },
+  },
+  ({ state }) => (
+    <>
+      <Switch>
+        <Route exact path='/pools/'>
+          <div>
+            {state.pools !== undefined ? (
+              <ul>
+                {state.pools.valueSeq().map(pool => (
+                  <li key={pool.$id}>
+                    <Link to={location => `${location.pathname}/${pool.$id}`}>{pool.name_label}</Link>
+                  </li>
+                ))}
+              </ul>
+            ) : (
+              <p>Loading objects ...</p>
```

Comment:
Is this not internationalized because the page itself is temporary?

Reply:
Yes.